File: src/lib/scala/Slate.Common/src/main/scala/slate/common/Credentials.scala
/**
<slate_header>
  author: Kishore Reddy
  url: https://github.com/kishorereddy/scala-slate
  copyright: 2015 Kishore Reddy
  license: https://github.com/kishorereddy/scala-slate/blob/master/LICENSE.md
  desc: a scala micro-framework
  usage: Please refer to license on github for more info.
</slate_header>
  */

package slate.common

case class Credentials(
                           id    : String = "",
                           name  : String = "",
                           email : String = "",
                           key   : String = "",
                           env   : String = "",
                           region: String = ""
                           )
{
}

File: src/common/common/src/main/kotlin/kiit/common/info/Credentials.kt
/**
 * <slate_header>
 * url: www.slatekit.com
 * git: www.github.com/code-helix/slatekit
 * org: www.codehelix.co
 * author: Kishore Reddy
 * copyright: 2016 CodeHelix Solutions Inc.
 * license: refer to website and/or github
 * about: A tool-kit, utility library and server-backend
 * mantra: Simplicity above all else
 * </slate_header>
 */

package kiit.common.info

data class Credentials(
    @JvmField val id: String = "",
    @JvmField val name: String = "",
    @JvmField val email: String = "",
    @JvmField val key: String = "",
    @JvmField val env: String = "",
    @JvmField val region: String = "",
    @JvmField val roles: String = ""
) {
    companion object {
        @JvmStatic
        val empty = Credentials()
    }
}
